depfinder/main.py

```diff
@@ -59,6 +59,18 @@
 AST_QUESTIONABLE = tuple(list(AST_TRY) + [ast.FunctionDef, ast.ClassDef])
 del AST_TRY
 
+PACKAGE_NAME = None
+
+_HERE = os.path.dirname(os.path.abspath(__file__))
+with open(os.path.join(_HERE, 'pkg_data', 'pkg_data.yml')) as f:
+    pkg_data = yaml.load(f)
```

Comment:
Did you consider using `pkgutil`? 

e.g.:
```python
In [2]: import yaml

In [3]: import pkgutil

In [4]: yaml.load(pkgutil.get_data("depfinder", "pkg_data/pkg_data.yml").decode())
Out[4]:
{'_PACKAGE_MAPPING': {'av': 'pyav',
  'bs4': 'beautifulsoup4',
  'cv2': 'opencv',
  'IPython': 'ipython',
  'netCDF4': 'netcdf4',
  'PIL': 'pillow',
  'skimage': 'scikit-image',
  'sklearn': 'scikit-learn',
  'stdlib_list': 'stdlib-list',
  'yaml': 'pyyaml'},
 '_NAMEDSPACE_MAPPING': {'ruamel.yaml': 'ruamel.yaml'},
 '_FAKE_PACKAGES': {'matplotlib': {'mpl_toolkits'},
  'pymongo': {'bson', 'gridfs'}}}
```

Reply:
Nope. I usually copy-n-paste that pattern and I never tried it. It is definetly better. I'll send another PR soon. Thanks!

Reply:
I suspect that

```python
pkgutil.get_data("depfinder", "pkg_data/pkg_data.yml"
```

should become

```python
pkgutil.get_data(__name__, "pkg_data/pkg_data.yml"
```

if the code is meant to be used in the same package.

> I'll send another PR soon

Sounds good. But no rush. Or I'll do it at some point :)  Thanks for all the contributions so far!